src/content/blog/2023-01-10-eslint-community-org.md

```diff
@@ -0,0 +1,81 @@
+to better organize that ecosystem. As a result, we created the
+`@eslint-community` GitHub organization as a home for high-value projects in the
+ESLint ecosystem.
+
+As you can read in the
+['`@eslint-community` GitHub organization' RFC](https://github.com/eslint/rfcs/tree/main/designs/2022-community-eslint-org),
+the goal of this new org is to have a place where community members can help
+ensure widely depended upon ESLint related packages live and thrive & never fall
+out of maintenance.  
+We will make sure these packages stay up to date with newer ESLint releases, and
+they don't hold the wider community back without depending on one person's
+GitHub/`npm` account.
+
+You might have noticed that some of your most beloved ESLint-related packages
+have already been moved over or forked by to the all new org. A full list of the
+ones that are already included are:
+
+* [`eslint-formatter-codeframe`](https://github.com/eslint-community/eslint-formatter-codeframe)
+  &
+  [`eslint-formatter-table`](https://github.com/eslint-community/eslint-formatter-table)  
+  We took these over from `@fregante](https://github.com/fregante)
+* [`eslint-plugin-eslint-plugin`](https://github.com/eslint-community/eslint-plugin-eslint-plugin)  
+  [@not-an-aardvark](https://github.com/not-an-aardvark) transferred this repo
+  to the new org, but is still maintaining it
+* [`eslint-plugin-promise`](https://github.com/eslint-community/eslint-plugin-promise)  
+  We took this one over from [@xjamundx](https://github.com/xjamundx)
+* [`eslint-plugin-security`](https://github.com/eslint-community/eslint-plugin-security)  
+  We took this one over from the
+  [@nodesecurity](https://github.com/nodesecurity) team, but
+  [@nzakas](https://github.com/nzakas) is still maintaining it
+* [`eslint-utils`](https://github.com/eslint-community/eslint-utils) &
+  [`regexpp`](https://github.com/eslint-community/regexpp)  
+  We created a fork from [@mysticatea](https://github.com/mysticatea)'s repos &
+  published them under the
+  [`@eslint-community` `npm` organization](https://npmjs.com/org/eslint-community).
+
+We intend to also create a (clean) fork, bring them up to date & publish them on
+`npm` as well for the following repos from
+[@mysticatea](https://github.com/mysticatea):
+
+* [`@mysticatea/eslint-plugin`](https://github.com/mysticatea/eslint-plugin)
+* [`eslint-plugin-es`](https://github.com/mysticatea/eslint-plugin-es)  
+  We'll work on getting all
+  [`eslint-plugin-es-x`](https://github.com/eslint-community/eslint-plugin-es-x)
+  features merged as well
+* [`eslint-plugin-eslint-comments`](https://github.com/mysticatea/eslint-plugin-eslint-comments)
+* [`eslint-plugin-node`](https://github.com/mysticatea/eslint-plugin-node)  
+  We'll work on getting all
+  [`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n)
+  features merged as well
+
+Of course, we don't want to stop there.  
```

Comment:
This is a good start, but we should have specific criteria that is published both in this blog post and somewhere in the org for people to refer to. It might make sense to create a repo that contains this info in a README and then use an issue template to allow people to apply to be included.

We need the details on what is expected after a package is transferred over, how to manage releases, who will be maintaining it, etc., to make this more appealing to others.